src/advanced/node-management.md

```diff
@@ -295,7 +295,7 @@ Response is a JSON object with the following fields:
 }
 ```
 
-### Private endpoints
+## Private endpoints
```

Comment:
Please make header levels the same (for Private endpoints and Public endpoints). IMO level 2 is better.